view/frontend/web/js/internals/common.js

```diff
@@ -98,15 +98,26 @@ define(['jquery', 'algoliaBundle'], function ($, algoliaBundle) {
         return check;
     };
 
-    window.getCookie = function (name) {
-        var value = "; " + document.cookie;
-        var parts = value.split("; " + name + "=");
-        if (parts.length == 2) {
-            return parts.pop().split(";").shift();
+    window.getCookie = function(name) {
+        var cookie,
```

Comment:
While this works (and mirrors the `Magento_PageCache/js/form-key-provider` implementation) for new code introductions it has been my preference that we start adhering to modern JS including the `let`/`const` convention instead.